docs/index.rst

```diff
@@ -6,8 +6,98 @@
 The Zoo
 =======
 
+A smart service catalog providing an overview of your services' development and operations.
+
+Adding custom checks
+====================
+
+If you want to add your own checks to the zoo, you need to create your own project with a python package in a directory specified by the :code:`ZOO_AUDITING_ROOT` environment variable. By default, the variable is pointing to :code:`zoo/auditing/standards`. (Don't forget that as for any other python package all folders need to have :code:`__init__.py` file.)
+
+The package needs to have two directories:
+
+- metadata folder with :code:`yml` files holding the list of errors code can trigger
+- checks folder with python :code:`check_` functions that actually perform the checks
+
+Both of them need to be *importable by the zoo* and the metadata and checks filenames have to match (the zoo imports the metadata files based on the python check filename).
+
+Metadata
+--------
+
+Metadata is a yaml file with a list of errors your checks yield. There are two compulsory parameters in the header:
+
+- :code:`namespace`
+- :code:`category`
+
+Then you list the errors with parameters:
+
+- :code:`id` - a unique identifier (*required*)
+- :code:`title` - a human-readable title (*required*)
+- :code:`description` - additional information about the error (*optional*)
+
+  - :code:`{details}` - object you can fill with information when yielding the error. Then you pass the :code:`{details}` object to the description where it is rendered (*optional*)
+
+**Metadata Structure:**
+
+.. literalinclude:: ..\zoo\auditing\standards\dummy_standards\metadata\check_constantly.yml
+    :language: yaml
+
+**Real life example:**
+
+.. code-block:: yaml
+
+    namespace: coafile
+    category: Configuration of coala
+    ---
+    - id: create_coafile
+      severity: warning
+      effort: high
+      title: Use coala to run static analysis checks on your repo
+      description: |
+        Add a `.coafile` so that people can see what rules and checks they should follow
+        when contributing to your repository,
+        and run [coala](https://coala.io) to find and fix these issues automatically.
+
+Checks
+------
+
+The zoo starts all functions in this file that have the prefix :code:`check_` (if you need any helper function just name them anything else).
```

Comment:
I'd change `The zoo starts all functions` with `The zoo detects all functions` as the discovery is decoupled from the execution.